utils.go

```diff
@@ -57,18 +57,18 @@ func BuildUri(otpType, secret, accountName, issuerName, algorithm string, initia
 	if otpType == OtpTypeHotp {
 		q.Set("counter", fmt.Sprintf("%d", initialCount))
 	}
-	u := url.URL {
-		Scheme: "otpauth",
-		Host: otpType,
-		Path: label,
+	u := url.URL{
+		Scheme:   "otpauth",
+		Host:     otpType,
+		Path:     label,
 		RawQuery: q.Encode(),
 	}
 	return u.String()
 }
 
 // get current timestamp
 func currentTimestamp() int {
-	return int(time.Now().Unix())
+	return int(time.Now().UTC().Unix())
```

Comment:
No need for extra UTC() call here, since Unix is already at UTC. Please refer to [go documentation](https://pkg.go.dev/time#Time.Unix) or [wikipedia](https://en.wikipedia.org/wiki/Unix_time).